http/src/main/resources/reference.conf

```diff
@@ -192,6 +194,8 @@ vpc {
     # Allows Leonardo proxy traffic on port 443
     {
       name-prefix = "leonardo-allow-https"
+      # See https://github.com/DataBiosphere/terra-resource-buffer/blob/master/src/main/java/bio/terra/buffer/service/resource/flight/CreateFirewallRuleStep.java#L37
+      rbs-name = "leonardo-ssl"
```

Comment:
optional nit, not sure which one is better `rbs-name` or `firewall-rule-name`.  RBS is bit confusing, and I don't think most people know what is RBS.

Reply:
I want to make it clear this is the name from RBS tho....`firewall-rule` is a bit redundant becuz this is inside a firewall rule config definition....like for `name-prefix` we don't say `firewall-rule-name-prefix`